client/src/pages/LogInForm.js

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable no-trailing-spaces */
+/* eslint-disable no-unused-vars */
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+
+function LogInForm(){
+    const [emailInput , setEmailInput]=useState("");
+    const [passwordInput , setPasswordInput]=useState("") ;
+    const [emailError , setEmailError]=useState("") ;
+    const [passwordError , setPasswordError]=useState("");
```

Comment:
Add space before and after equal sign.
Remove space before comma.
Remove space before semicolon